### ISSUE TITLE
[BUG] npm not showing up in vs code, npm not running, and npm not working in general -> newest version on mac

### ISSUE THREAD

**Issue by kyegomez** [npm/cli#5561]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

 npm install

up to date, audited 1423 packages in 3s

149 packages are looking for funding
  run `npm fund` for details

3 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

npm install @slack/bolt

changed 2 packages, and audited 1398 packages in 3s

149 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

npm run
Lifecycle scripts included in defalt@1.0.0:
  test
    echo "Error: no test specified" && exit 1

### Expected Behavior

node_modules appearing in directory

npm run....

Bolt app is running.

### Steps To Reproduce

1. In this environment...
2. With this config...
3. Run '...'
4. See error...

### Environment

- npm:8.15.0
- Node.js:v16.17.0
- OS Name: MAC OS 
- System Model Name: Macbook
- npm config:
``npm config ls
; node bin location = /usr/local/bin/node
; node version = v16.17.0
; npm local prefix = /Users/defalt
; npm version = 8.15.0
; cwd = /Users/defalt/Desktop/Athena-redirect-url
; HOME = /Users/defalt
```

**Comment by kyegomez** [comment]
I've deleted node and npm multiple ways including both with brew and from node website.

**Comment by fritzy** [comment]
It looks like your cwd `/Users/defalt/Desktop/Athena-redirect-url` does not have a package.json in it, so it's using one from `/Users/defalt`.

**Comment by kyegomez** [comment]
Doesn't
`npm install`
build one at run time?

Or would I have to build it myself?

**Comment by ljharb** [comment]
@APACOWL no, `package.json` must always exist - you create and manage it.

**Comment by APACOWL** [comment]
```
Lifecycle scripts included in athena@1.0.0:
  test
    npm run
```
now i keep getting this error and I can't run using 
`npm run`

and when i use node app.js

```
Error: Cannot find module './listeners/app-home/authedView'
Require stack:
- /Users/defalt/Downloads/oauth-sample-part3-auth/listeners/utils/home_tab_callbacks.js
- /Users/defalt/Downloads/oauth-sample-part3-auth/listeners/events/app-home-opened.js
- /Users/defalt/Downloads/oauth-sample-part3-auth/listeners/events/index.js
- /Users/defalt/Downloads/oauth-sample-part3-auth/listeners/index.js
- /Users/defalt/Downloads/oauth-sample-part3-auth/app.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/defalt/Downloads/oauth-sample-part3-auth/listeners/utils/home_tab_callbacks.js:2:20)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/defalt/Downloads/oauth-sample-part3-auth/listeners/utils/home_tab_callbacks.js',
    '/Users/defalt/Downloads/oauth-sample-part3-auth/listeners/events/app-home-opened.js',
    '/Users/defalt/Downloads/oauth-sample-part3-auth/listeners/events/index.js',
    '/Users/defalt/Downloads/oauth-sample-part3-auth/listeners/index.js',
    '/Users/defalt/Downloads/oauth-sample-part3-auth/app.js'
  ]
}
```
